src/ThemeProvider.tsx

```diff
@@ -146,17 +146,18 @@ function chooseColorScheme(colorMode: ColorMode, dayScheme: string, nightScheme:
   }
 }
 
-function applyColorScheme(theme: Theme, colorScheme: string) {
+function applyColorScheme(theme: ThemePrevalType, colorScheme: string): Theme {
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
   if (!theme.colorSchemes) {
-    return theme
+    return theme as Theme
   }
```

Comment:
Figuring out the case here - is this a real scenario? (does theme-preval copy a colorscheme by default - still discovering)